app/models/emoji.rb

```diff
@@ -60,6 +60,7 @@ def self.clear_cache
     Discourse.cache.delete(cache_key("standard_emojis"))
     Discourse.cache.delete(cache_key("aliases_emojis"))
     Discourse.cache.delete(cache_key("all_emojis"))
+    Discourse.cache.delete(cache_key("tonable_emojis"))
```

Comment:
At some point we should start having an array of the cache keys and iterate over them and call `Discourse.cache.delete`